newman/localhost/hmda-filing/bank0/run_newman.sh

```diff
@@ -0,0 +1,12 @@
+#! /bin/bash
+
+if (( $# < 2 ))
+  then
+    echo "usage: ./run_newman.sh <num_times> <num_rows>"
+    exit 1
+fi
+
+for i in `seq 0 $1`
+do
+  eval 'newman run hmda-filing-api-test-'$2'.json -d hmda-filing-2018-config-bank0-'$2'.json'
```

Comment:
We should add another variable to select bank

Reply:
Created ticket for this https://github.com/cfpb/hmda-platform/issues/3194